core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/WorkflowCompiler.scala

```diff
@@ -72,10 +156,11 @@ class WorkflowCompiler(
       opResultStorage: OpResultStorage,
       executionStateStore: ExecutionStateStore
   ): Workflow = {
+    // TODO: remove this function after separating compiler as a standalone service
```

Comment:
for those comments describing future removal, instead of adding todos in the first line of the function, better to make it a block comment before the function signature. also try to use `deprecated` (@Deprecated) to mark content to be removed.

Reply:
done